packages/appeals-service-api/src/routes/v2/users/_id/appeals/service.js

```diff
@@ -33,6 +33,13 @@ async function getAppealsForUser(userId) {
 		.map((appealToUser) => appealToUser.Appeal?.legacyAppealSubmissionId)
 		.filter(filterNotNull);
 
+	// find v2 draft submissions
+	const v2Drafts = user.Appeals.filter(
+		(appealToUser) =>
+			appealToUser.Appeal?.AppellantSubmission?.submitted === false &&
+			appealToUser.Appeal?.AppealCase === null
```

Comment:
do we need the second line? won't submitted be enough?

Reply:
Yes you're correct. 

I had added the second line as the first appeal submission that was added when first building the journey was attached to an appeal that had the appeal case published. Obviously going through the complete journey normally this wouldn't happen.

Ive made that first submission draft as submitted = true and removed the appeal case check.